### PR TITLE
improvement(gradual test): warm up commands as parameter

### PR DESCRIPTION
**Pull request by juliayakovlev**
Warm up cassandra-stress commands should be defined as parameter in the test yaml.

### Testing

- [ ]

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: performance_regression_gradual_grow_throughput.py [hunk 1]
```diff
@@ -18,24 +19,58 @@ class CSPopulateDistribution(Enum):
...
         self.REPLICATION_FACTOR = 3  # pylint: disable=invalid-name
 
+    def throttle_steps(self, workload_type):
+        throttle_steps = self.params["perf_gradual_throttle_steps"]
+        if workload_type not in throttle_steps:
+            TestFrameworkEvent(source=self.__class__.__name__,
+                               message=f"Throttle steps for '{workload_type}' test is not defined in "
+                                       f"'perf_gradual_throttle_steps' parameter",
+                               severity=Severity.CRITICAL).publish()
+        return throttle_steps
```
Comment (fruch) [inline performance_regression_gradual_grow_throughput.py:54]: isn't this should be ?
```suggestion
        return throttle_steps[workload_type] 
```

File: performance_regression_gradual_grow_throughput.py [hunk 2]
```diff
@@ -186,30 +232,14 @@ def run_gradual_increase_load(self, stress_cmd_templ, stress_num, num_loaders, c
         with open(filename, "w", encoding="utf-8") as res_file:
             res_file.write(total_summary_json)
 
-        screenshots = self.monitors.get_grafana_screenshots(self.monitors.nodes[0], self.start_time)
```
Comment (fruch) [inline performance_regression_gradual_grow_throughput.py:189]: we don't generate the overwall screenshots anymore ? 

can you clarify why you are remove this this ?

File: performance_regression_gradual_grow_throughput.py (hunk 1 above)
Comment (juliayakovlev) [reply performance_regression_gradual_grow_throughput.py]: right, fixed

File: performance_regression_gradual_grow_throughput.py (hunk 2 above)
Comment (juliayakovlev) [reply performance_regression_gradual_grow_throughput.py:189]: It is part of `self.update_test_details(scylla_conf=True)` that called in each step

**Review by fruch — DISMISSED** [review]
LGTM

**Comment by juliayakovlev** [comment]
Closed with https://github.com/scylladb/scylla-cluster-tests/pull/8836 (I broke this branch)

**Pull request closed without merging**